src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -61,10 +61,12 @@
       SAMPLE_METHOD_TO_VIEW_TRANSFORMER_MAP =
           ImmutableMap.of(
               JAVA, JavaSampleMethodToViewTransformer.class,
+              GO, GoSampleMethodToViewTransformer.class,
```

Comment:
nit: What's your criterion for the order in which you're declaring these constants? I'd just order alphabetically by key so it's very easy to visually locate a given language.